packages/cli/__tests__/main.ts.unit.test.ts

```diff
@@ -29,7 +29,7 @@ describe("behavior of main.ts", () => {
 
         it("should exit with non zero return code if Imperative.init() fails - calling main.ts directly", async () => {
 
-            const mainTs = resolve(__dirname, '../../packages/cli/src/main');
+            const mainTs = resolve(__dirname, '../lib/main.js');
```

Comment:
I guess I had a similar question for `packages/cli/src/main.ts` when we specify the `imperative.js` instead of just `imperative`.
I think I assumed it was "ok" since things will eventually be pulled from there anyways but definitely a good question 😋